Added my name/GitHub link to the Contributors section on the About page.

CoderAJ7470 opened the pull request:
<!-- Provide a general summary of the pull request in the Title above -->
<!-- Thank you for taking the time to help us improve -->
<!-- Below is a list of items we ask you include with your pull request -->

## Related Issue
<!-- Each Pull Request must be related to an issue, please reference the issue -->
<!-- If suggesting a new feature, please open an issue to discuss before submitting a pull request -->
<!-- Example - "Resolves #1" -->
Resolves #

### Description
<!-- Describe your changes in detail here, utilize a list to show impacted files where needed -->


### Checklist
<!-- Please make sure all items are checked before submitting a pull request -->
<!-- Place a 'x' in the brackets to represent the item as done -->
- [x] I have read and understand the **CODE OF CONDUCT** as outlined
- [x] I have read the **CONTRIBUTING** document
- [x] I have validating my HTML as outlined in the **CONTRIBUTING** document
- [x] My repository is up to date, and merge conflicts have been resolved
- [x] This pull request relates to an open issue and has been tagged above

### Who needs to review this change?
<!-- Do you have someone specific in mind to review the issue, list them below -->
<!-- Example - "@mstub can you take a look at this?" -->

mstub commented on docs/about.html:
```diff
@@ -102,6 +102,7 @@ <h2 id="organizers-volunteers">Contributors</h2>
                         <li><a href="https://github.com/stu7" target="_blank" title="Norman Haase" rel="external">Norman Haase</a></li>
                         <li><a href="https://github.com/Jason9199" target="_blank" title="Jason Carpenter" rel="external">Jason Carpenter</a></li>
                         <li><a href="https://github.com/brycejech" target="_blank" title="Bryce Jech" rel="external">Bryce Jech</a></li>
+						<li><a href="https://github.com/coderaj7470" target="_blank" title="Ameya Joshi" rel="external">Ameya Joshi</a></li>
```
could you remove the extra whitespace here @CoderAJ7470 and bring it in line with the other list items? thanks!

CoderAJ7470 commented:
Hello Matt,

I have deleted the whitespace that was appearing at the start of the link I added for my name/GitHub profile. It should be corrected now. :)
